pdf/model/writer.go

```diff
@@ -55,6 +77,57 @@ func SetPdfCreator(creator string) {
 	pdfCreator = creator
 }
 
+func getPdfKeywords() string {
+	return pdfKeywords
+}
+
+// SetPdfKeywords sets the Keywords attribute of the output PDF.
+func SetPdfKeywords(keywords string) {
+	pdfKeywords = keywords
+}
+
+func getPdfModifiedDate() time.Time {
+	return pdfModifiedDate
+}
+
+// SetPdfModifiedDate sets the ModifiedDate attribute of the output PDF.
```

Comment:
I believe it's supposed to say `the ModDate attribute`

Reply:
Indeed. Fixed